packages/common/src/errors.ts

```diff
@@ -66,10 +78,12 @@ export function errorToPlainObject<T extends Error>(error: T) {
     };
 }
 
+/**
+ * Checks if the `error` is an object compatible with the Error interface; that is,
+ * it has properties 'name' and 'message' of type string. The object could be an
+ * instance of an Error, or it could be some other kind of object that has these
+ * properties.
+ */
 export function isErrorLikeObject(error: unknown): error is Error {
-    return (
-        isPlainObject(error) &&
-        typeof (error as { name?: string }).name === 'string' &&
-        typeof (error as { message?: string }).message === 'string'
-    );
+    return isObject(error) && typeof error.name === 'string' && typeof error.message === 'string';
```

Comment:
Maybe use in?

Reply:
And verify stack is available without dereferencing it?

Reply:
`stack` is an optional property.

What do you mean by "use in"?
```ts
declare const x: unknown;
'hello' in x; // type error: 'x' is of type 'unknown'.
```

Reply:
well, since stack is optional this is irrelevant. I thought it's required.
I meant verifying that stack is there as well, and using `'stack' in error` (instead of `error.stack`) to verify this without generation of the stack.